src/snyk/common/configuration/configuration.ts

```diff
@@ -171,13 +171,7 @@ export class Configuration implements IConfiguration {
       return this.processEnv.SNYK_VSCE_DEVELOPMENT_SNYKCODE_BASE_URL;
     } else if (this.customEndpoint) {
       const url = new URL(this.customEndpoint);
-
-      if (Configuration.isSingleTenant(url)) {
-        url.host = url.host.replace('app', 'deeproxy');
-      } else {
-        url.host = `deeproxy.${url.host}`;
-      }
-      url.pathname = url.pathname.replace('api', '');
+      url.host = url.host.replace('api', 'deeproxy');
```

Comment:
I could be missing something but in this case the `customEndpoint` will be exactly what's provided in the UI, which is validated anyway and can only be the new format (api.snyk.io). And so I don't think there's any situation in which the old format will be provided?

Reply:
This function actually was not used anymore so I removed it from the code but the same answer could apply to https://github.com/snyk/vscode-extension/pull/466#discussion_r1627670015 I think. Let me know what you think!

Reply:
Hm, what about people already having configured a custom endpoint? They would see a validation error without having changed anything. This may already happen, now, but I'd love to improve the UX.

Reply:
I agree, in hindsight, that the validation is very inconvenient for customers who have the old endpoint configured. In https://snyksec.atlassian.net/browse/IDE-126 we said we want to enforce the new endpoint to be used, is there some other way we can enforce it while not breaking existing customer's setup?